script/Ciel/Day 10/10_01_CIEL10_3.txt

```diff
@@ -78,68 +89,82 @@ I raise my head at the sound of her shy voice.
-Umm... So, you're saying that you don't mind, no matter what kind of body I possess?"
+So, um...
+ Does that mean you wouldn't mind no matter what kind of body I had, Tohno-kun?"
 }
 [sha:60fcee72d5ff2ce0acc64a7ff2342f61ce76d984]{
 -- Page 134, Offset 27421.
 -- 「もちろんです。……まあ、どんな体、という意味を広義に解釈するとそのかぎりではないですけどね」
 // Keep a legalese/stiff tone when rephrasing this
-"Of course! Though... To the exception of a broad interpretation definition of 'body'."
+"Of course!
+ Well... assuming we don't get %{i}too%{/i} liberal with the definition of 'body'."
 }
 [sha:bf649d3a615dd9844dec4f2c5167239c002bf02c]{
 -- Page 134, Offset 27422.
 -- 「……じゃあ、一緒にいても嫌わない、ですか？」
-"...So, you wouldn't hate me if we were together, would you?"
+"Then... you don't hate being around me, either?"
 }
 [sha:8145fbce61d481bff8dfaf7c7fbbb15aad1b2a78]{
 -- Page 135, Offset 27423.
 -- 「嫌いません。むしろ謝らせてください。
-"I wouldn't. On the contrary, allow me to apologize.
+"Not one bit.
+ On the contrary, allow me to apologize.
 }
 [sha:a5f693e655f8b2adac69f778c4e9c7df543469f8]{
 -- Page 135, Offset 27424.
 -- 昨日は一日、先輩を困らせてばかりでした。アルクェイドの事だって、先輩に相談していれば穏便にすんだだろうし。
-Yesterday the whole day, I was a pain in the neck. The whole business with Arcueid would probably have ended peacefully if I had taken your advice.
+I was a total pain in the neck all day yesterday.
+ I mean, if I'd just listened to you, I'm sure that whole business with Arcueid would have ended peacefully.
 }
 [sha:5ad3fe8a9c3254b277165278aaf001be9939dd41]{
 -- Page 135, Offset 27425.
 -- だからホントにもう、全面的に俺が悪かったです。これからはちゃんと先輩の忠告を聞きますから、許してください」
-So, honestly, everything was my fault. From now on, I'll listen to your instructions properly, so please, forgive me, Senpai."
+// > これからはちゃんと先輩の忠告を聞きます
+// sure you will buddy
+So let me be clear, this is all 100% my fault.
+ I swear I'll listen to you from now on, Senpai, and I hope that you'll forgive me."
 }
 [sha:2ea8b759ca631cdc20d9e206518ccc305a29fbff]{
 -- Page 136, Offset 27426.
 -- 頭を下げて謝罪する。
-I bow my head in apology.
+I bow my head and apologize.
 }
 [sha:9ec5798a0998378072228ab617ac59032b2a274e]{
 -- Page 136, Offset 27427.
 -- 経緯はどうあれ俺は先輩を悲しませた。二度とあんな顔をさせないよう、自分の胸に刻みつける。
-Regardless of how it happened, I made Senpai sad. I engrave in my heart the resolve never to bring her sorrow ever again.
+I made Senpai sad.
+ The circumstances around how it happened don't matter――there's still no excuse.
+ I vow to myself that I'll never make her feel that way again.
 }
 [sha:c4f6fa2c299c58661e70603dc818c9582cf3f3da]{
 -- Page 137, Offset 27428.
 -- 「……まったく。遠野くんには<敵|かな>いませんね。忠告を聞くって、まだ戦う気満々じゃないですか。
-"Goodness gracious... There's just no winning against you, Tohno-kun. 'Listening to my instructions'? It sounds to me like you're still spoiling for a fight.
+// まだ戦う気満々 as in listening to her [on the battlefield] -> he still wants to be involved?
+"Goodness gracious...
+ There's just no winning against you, Tohno-kun.
+ You'll listen to me from now on?
+ Sounds like you still plan on diving into the fray.
```

Comment:
Not sure, when I first read it sounded like "You say that, but you're still fully intent on fighting, aren't you?"
But on further inspection its a bit ambigious.